src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -55,24 +55,27 @@
   <!-- ********************************************************** -->
   <!-- Sonar C#/VB analyzers -->
   <!-- ********************************************************** -->
+  <!-- We assume that the package restore has happened so the files are already on disc -->
   <Import Project="..\EmbeddedSonarAnalyzer.props" />
   <ItemGroup Label="Embedded SonarLint analyzers">
     <!-- Reference the packages we want to import, but don't include them as analyzers for this project -->
     <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
     <PackageReference Include="SonarAnalyzer.VisualBasic" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
 
-    <!-- Select all of the analyzer binaries (we assume that the package restore has happened so the files are already on disc) -->
-    <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
-    <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.VisualBasic\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
+    <!-- Include all of the analyzer binaries including third-party binaries in the VSIX -->
```

Comment:
Explicitly split out (A) the set of files we want to include in the VSIX and (B) the set of files we want to include the AsmRef report.

We don't need to copy set (A) to the output directory; it's enough to add them to the `VSIXSourceItem` group.

This now does what we want:
* the correct versions are included in the VSIX
* the expected files are included in the AsmRef check
* the correct NewtonSoft version is used when running the tests locally

We do need to copy set (B) to the output directory; the AsmRef report is based on the files in that directory.

The reason the tests were failing locally is because the test project was copying the v13 version of the NuGet assembly from the VSIX bin folder. I don't know why it wasn't also failing on the CI machine.